packages/syft/src/syft/service/project/project.py

```diff
@@ -1180,19 +1182,16 @@ def exchange_routes(
 
     def add_request(
         self,
-        request: Request,
-    ):
-        request_event = ProjectRequest(request=request)
-        result = self.add_event(request_event)
-
-        if isinstance(result, SyftSuccess):
-            return SyftSuccess(message="Request created successfully")
-        return result
+        *args: Request,
+    ) -> None:
```

Comment:
With this we can basically do `project.add_request(request1, request2, request3, ...)`